NEMO/templates/calendar/reservation_event_feed.html

```diff
@@ -7,6 +7,12 @@
 		{% elif personal_schedule and x.area %}
 			"title": "{{ x.title|default:x.reservation_item.name|escapejs }}",
 			"color": "#84CD84",
+		{% elif all_tools and x.tool %}
```

Comment:
you are changing the tooltip here which is good.
I am wondering if the title of the event should be the tool name rather than the user. since the view is called "all tools" it might make more sense to see the tool name, and hover for user details (just like in the personal schedule view)

Reply:
This is now the tool name